src/PageLayout/PageLayout.tsx

```diff
@@ -0,0 +1,363 @@
+import React from 'react'
+import {Box} from '..'
+import {BetterSystemStyleObject, merge, SxProp} from '../sx'
+
+const REGION_ORDER = {
+  header: 0,
+  paneStart: 1,
+  content: 2,
+  paneEnd: 3,
+  footer: 4
+}
+
+const SPACING_MAP = {
+  // none: '0', Should `none` be an option?
+  condensed: 3,
+  normal: [3, null, null, 4]
+}
+
+const PageLayoutContext = React.createContext<{
+  outerSpacing: keyof typeof SPACING_MAP
+  rowGap: keyof typeof SPACING_MAP
+  columnGap: keyof typeof SPACING_MAP
+}>({
+  outerSpacing: 'normal',
+  rowGap: 'normal',
+  columnGap: 'normal'
+})
+
+// ----------------------------------------------------------------------------
+// PageLayout
+
+export type PageLayoutProps = {
+  /** The maximum width of the page container */
+  containerWidth?: keyof typeof containerWidths
+  /** The spacing between the outer edges of the page container and the viewport */
+  outerSpacing?: keyof typeof SPACING_MAP // Should this be called `padding`?
+  rowGap?: keyof typeof SPACING_MAP
+  columnGap?: keyof typeof SPACING_MAP
+} & SxProp
+
+const containerWidths = {
+  full: '100%',
+  medium: '768px',
+  large: '1012px',
+  xlarge: '1280px'
+}
+
+// TODO: refs
+const Root: React.FC<PageLayoutProps> = ({
+  containerWidth = 'xlarge',
+  outerSpacing = 'normal',
+  rowGap = 'normal',
+  columnGap = 'normal',
+  children,
+  sx = {}
```

Comment:
Should sx have a default value?

Reply:
Needed this to make TypeScript happy with the `merge()` function